astropy/coordinates/baseframe.py

```diff
@@ -341,14 +341,18 @@ def __new__(cls, reprname, framename, defaultunit='recommended'):
         sets the expected input representation class, thereby changing the
         expected keyword arguments for the data passed in. For example, passing
         ``representation_type='cartesian'`` will make the classes expect
-        position data with cartesian names, i.e. ``x, y, z`` in most cases.
+        position data with cartesian names, i.e. ``x, y, z`` in most cases
+        unless overriden via `frame_specific_representation_info`.
+        Check out `<this frame>().representation_info`.
```

Comment:
This makes the docs fail:

```
/home/circleci/project/build/lib.linux-x86_64-3.6/astropy/coordinates/builtin_frames/altaz.py:docstring of astropy.coordinates.AltAz:40:py:obj reference target not found: <this frame>().representation_info
```

(https://circleci.com/gh/astropy/astropy/38707)